Description: Fix build failure by using linux/videodev2.h instead of
  deprecated linux/videodev.h
Orign: ubuntu
Author: Daniel Polehn <dpolehn@gmail.com>
Ubuntu-Bug: https://bugs.launchpad.net/ubuntu/+source/uvccapture/+bug/756023
Debian-Bug: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=621963

Index: uvccapture/v4l2uvc.c
===================================================================
--- uvccapture.orig/v4l2uvc.c	2008-07-31 22:26:18.000000000 -0400
+++ uvccapture/v4l2uvc.c	2012-06-29 12:48:31.912949668 -0400
@@ -28,7 +28,7 @@
 #include <errno.h>
 #include <unistd.h>
 #include <fcntl.h>
-#include <linux/videodev.h>
+#include <linux/videodev2.h>
 #include <sys/mman.h>
 #include <sys/ioctl.h>
 #include "v4l2uvc.h"
Index: uvccapture/uvccapture.c
===================================================================
--- uvccapture.orig/uvccapture.c	2008-07-31 22:26:18.000000000 -0400
+++ uvccapture/uvccapture.c	2012-06-29 12:48:31.912949668 -0400
@@ -31,7 +31,7 @@
 #include <unistd.h>
 #include <jpeglib.h>
 #include <time.h>
-#include <linux/videodev.h>
+#include <linux/videodev2.h>
 
 #include "v4l2uvc.h"
 
